app/models/organization.rb

```diff
@@ -264,6 +264,11 @@ def earliest_reporting_year
     year
   end
 
+  def last_distribution_date
+    distribution = distributions.order(issued_at: :desc).first
```

Comment:
I would call this display_last_distribution_date  rather than last_distribution_date.   Naming it last_distribution_date implies that it is a date, to me, anyway.

Reply:
Sounds good to me. I've made this change and have pushed up the changes.